test/java/seedu/addressbook/testutil/PersonBuilder.java

```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.testutil;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Phone;
+
+/**
+ * Factory class for building TestPerson
+ */
+public class PersonBuilder {
+
+    private TestPerson person;
+
+    public PersonBuilder() {
+        this.person = new TestPerson();
+    }
+
+    public PersonBuilder withName(String name) throws IllegalValueException {
+        this.person.setName(new Name(name));
+        return this;
+    }
+
+    public PersonBuilder withPhone(String phone, boolean isPrivate) throws IllegalValueException {
```

Comment:
Okay. Implemented two methods for each field. 
For example withPublicPhone and withPrivatePhone.